### DISCUSSION TITLE
about hp_search

### DISCUSSION THREAD

**Discussion by kongbia** [STVIR/pysot#463] [Q&A]
请问在搜索超参时为什么每一次开始 需要重新构建tracker？

` cfg.TRACK.PENALTY_K = float(pk) `
`cfg.TRACK.WINDOW_INFLUENCE = float(wi)`
   `cfg.TRACK.LR = float(lr)`
   `cfg.TRACK.INSTANCE_SIZE = int(ins)`
   `# rebuild tracker`
   `tracker = build_tracker(model)`

**Comment by ZhiyuanChen** [comment]
大概是因为template变了，需要重新初始化
